test/custom_task_test.go

```diff
@@ -401,3 +403,130 @@ spec:
 		t.Fatalf("Failed to get PipelineRun `%s`: %s", pipelineRun.Name, err)
 	}
 }
+
+func applyController(t *testing.T) {
+	t.Log("Creating Wait Custom Task Controller...")
+	cmd := exec.Command("ko", "apply", "-f", "./config/controller.yaml")
+	cmd.Dir = "./wait-task"
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to create Wait Custom Task Controller: %s, Output: %s", err, out)
+	}
+}
+
+func cleanUpController(t *testing.T) {
+	t.Log("Tearing down Wait Custom Task Controller...")
+	cmd := exec.Command("ko", "delete", "-f", "./config/controller.yaml")
+	cmd.Dir = "./wait-task"
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to tear down Wait Custom Task Controller: %s, Output: %s", err, out)
+	}
+}
```

Comment:
the other tests in this file are "simulating behavior of runs" can we update them to use the wait custom task? 

this can be done in a follow up PR

Reply:
Yeah SG! What I can think of is there are two major things:
1. Test Custom Task in Pipeline
2. Test Custom Task Run with CRD
I'll open two prs for the two sub tasks above. wdyt?

Thanks for catching it!